integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Content metadata exporter for SAP SuccessFactors.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from logging import getLogger
+
+from integrated_channels.integrated_channel.exporters.content_metadata import ContentMetadataExporter
+
+
+LOGGER = getLogger(__name__)
+
+
+class SapSuccessFactorsContentMetadataExporter(ContentMetadataExporter):  # pylint: disable=abstract-method
+    """
+    SAP SuccessFactors implementation of ContentMetadataExporter.
+    """
+
+    DATA_TRANSFORM_MAPPING = {
+        'courseID': 'key',
+        'providerID': 'provider_id',
+        'status': 'status',
+        'title': 'title',
+        'description': 'description',
+        'thumbnailURI': 'image',
+        'content': 'launch_points',
+        'revisionNumber': 'revision_number',
+    }
+
+    def transform_provider_id(self, content_metadata_item):  # pylint: disable=unused-argument
+        """
+        Return the provider ID from the integrated channel configuration.
+        """
+        return self.enterprise_configuration.provider_id
+
+    def transform_status(self, content_metadata_item):  # pylint: disable=unused-argument
+        """
+        Return the status of the content item.
+        """
+        return 'ACTIVE'
+
+    def transform_title(self, content_metadata_item):
+        """
+        Return the title of the content item.
+        """
+        return [{
+            'locale': 'English',
```

Comment:
i think we had support before for checking the language of the course and adjusting this based on the language.